test/sdp.jl

```diff
@@ -1,4 +1,7 @@
 @testset "Semidefinite Programming" begin
+
```

Comment:
Please add tests to `constraint.jl` that test model generation. The proposed representation of SD constraints is not obviously compatible with the existing `constraintobject` interface, which suggests a design issue. Maybe we need an MOI set which is a full square matrix that includes imposition of equality constraints if necessary.

Reply:
Fixed in 1649b42e1c22944ebef3ee75dbcc1bbc4410a185.